script.js

```diff
@@ -0,0 +1,66 @@
+let word = 'APPLE'
+
+// Hide the word: 
+document.getElementById('word').innerHTML = word
+document.getElementById('word').style.display = 'none'
+
+// Check how many letters and write the _ :
+let underscore = word.length
+
+switch (underscore){
+    case '1':
+        document.getElementById('underscore').innerHTML = '_';
+        break;
+    case '2':
+        document.getElementById('underscore').innerHTML = '_ _';
+        break;
+    case '3':
+        document.getElementById('underscore').innerHTML = '_ _ _';
+        break;
+    case '4':
+        document.getElementById('underscore').innerHTML = '_ _ _ _';
+        break;
+    case '5':
+        document.getElementById('underscore').innerHTML = '_ _ _ _ _';
+        break;
+    case '6':
+        document.getElementById('underscore').innerHTML = '_ _ _ _ _ _';
+        break;
+    default:
+    alert('This word is too big');
+    break;
+}
+
+function tryLetter(){
+    //Save and transform to upper case the letter the user wrote:
+    let letter = document.getElementById('letter').value
+    let LETTER = letter.toUpperCase()
+
+    //Check if the letter is on the word:
+    let error = 0
+    
+    for (let i=0; i<word.length; i++){
```

Comment:
So then in this for loop, instead of looping over the word, you can loop over the array of individual letters
And see if there is a match
And only change to display block the letter that matched

I think this might also solve the problem with the error going 5 by 5 instead of 1 by 1